bika/lims/setuphandlers.py

```diff
@@ -582,20 +561,11 @@ def addColumn(cat, col):
         addColumn(bc, 'inactive_state')
         addColumn(bc, 'cancellation_state')
         addColumn(bc, 'getAnalysts')
-        addColumn(bc, 'getSampleID')
-        addColumn(bc, 'getRequestID')
-        addColumn(bc, 'getClientOrderNumber')
-        addColumn(bc, 'getClientReference')
-        addColumn(bc, 'getClientSampleID')
         addColumn(bc, 'getContactTitle')
         addColumn(bc, 'getClientTitle')
-        addColumn(bc, 'getSamplePointTitle')
-        addColumn(bc, 'getSampleTypeTitle')
         addColumn(bc, 'getAnalysisService')
-        addColumn(bc, 'getDatePublished')
         addColumn(bc, 'getDateReceived')
         addColumn(bc, 'getDateSampled')
```

Comment:
Is `getDateSampled` still necessary? I think this metadata column was only used for Sample objects

Reply:
@xispa I am not sure, isn't the following piece of code trying to retrieve the value of ```getDateSampled``` column of a reference sample? Or will this call reference sample ```getDateSampled()``` method?

https://github.com/naralabs/bika.lims/blob/a700344a93f78c297a9887d4e0f6c24306fbcb57/bika/lims/browser/templates/referencesample_view.pt#L71-L77